install_config/installing-operator-framework.adoc

```diff
@@ -101,6 +101,7 @@ labels, and more.
 
 include::modules/installing-olm-using-ansible.adoc[leveloffset=+1]
 include::modules/launching-your-first-operator.adoc[leveloffset=+1]
+include::modules/uninstalling-olm-using-ansible.adoc[leveloffset=+1]
```

Comment:
Installing and uninstalling are separate user stories. I suggest putting this content in its own assembly.